apps/ledger-live-desktop/src/newArch/features/Collectibles/Ordinals/components/Inscriptions/index.tsx

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+import { Account } from "@ledgerhq/types-live";
+import { Box, Flex } from "@ledgerhq/react-ui";
+import { useInscriptionsModel } from "./useInscriptionsModel";
+import TableContainer from "~/renderer/components/TableContainer";
+import TableHeader from "LLD/features/Collectibles/components/Collection/TableHeader";
+import { OrdinalsRowProps, TableHeaderTitleKey } from "LLD/features/Collectibles/types/Collection";
+import TableRow from "LLD/features/Collectibles/components/Collection/TableRow";
+import ShowMore from "LLD/features/Collectibles/components/Collection/ShowMore";
+import { MediaProps } from "LLD/features/Collectibles/types/Media";
+
+type ViewProps = ReturnType<typeof useInscriptionsModel>;
+
+type Props = {
+  account: Account;
+};
+
+export type InscriptionsItemProps = {
+  isLoading: boolean;
+  tokenName: string;
+  collectionName: string;
+  tokenIcons: OrdinalsRowProps["tokenIcons"];
+  media: MediaProps;
+  onClick: () => void;
+};
+
+const Item: React.FC<InscriptionsItemProps> = ({
+  isLoading,
+  tokenName,
+  collectionName,
+  tokenIcons,
+  media,
+  onClick,
+}) => {
+  return (
+    <TableRow
+      isLoading={isLoading}
+      tokenName={tokenName}
+      collectionName={collectionName}
+      tokenIcons={tokenIcons}
+      media={media}
+      onClick={onClick}
+    />
+  );
+};
+
+function View({ temporyObjects, displayShowMore, onShowMore }: ViewProps) {
```

Comment:
`temporyObjects` can be renamed as `mockedItems`

And then put all these mocked items in a separate file for test purpose later